src/frontend/staff-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -0,0 +1,186 @@
+import { LoggerService } from '@core/services/logger.service';
+import { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { ViolationTicket, OcrViolationTicket, Field, ViolationTicketCount } from 'app/api';
+import { DatePipe } from '@angular/common';
+
+export interface IViolationTicketService {
+  getAllOCRMessages(ocrViolationTicket: OcrViolationTicket);
+  setViolationTicketFromJSON(ocrViolationTicket: OcrViolationTicket, violationTicket: ViolationTicket): ViolationTicket;
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ViolationTicketService implements IViolationTicketService {
+  public ocrMapping: mappingOCR[] = [
+    { key: "ticket_number", heading: "Ticket Number" },
+    { key: "violation_date", heading: "Date" },
+    { key: "violation_time", heading: "Time" },
+    { key: "surname", heading: "Surname" },
+    { key: "given_names", heading: "Given name(s)" },
+    { key: "drivers_licence_province", heading: "Prov/State of DL" },
+    { key: "drivers_licence_number", heading: "DL Number" },
+  ];
+  public count1OcrMapping: mappingOCR[] = [
```

Comment:
Turning count to count array might be better although the maximum must be 3.  It can remove quiet a lot lines of code and prevent typo or errors.